PatternPal/PatternPal.Extension/ViewModels/CheckResultViewModel.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace PatternPal.Extension.ViewModels
 {
+    // TODO Comment
```

Comment:
@reviewer

Reply:
Didn't do since was a bit outside my knowledge, so documentation would be prefereable from @DanielvanDamme or @KnapSac